keyboards/numatreus/keymaps/hdbx/keymap.c

```diff
@@ -0,0 +1,230 @@
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY,
+    _HDBX,
+    _LOWER,
+    _RAISE,
+    _GAME,
+    _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  HDBX,
+  MCR1,                // マクロ1
+  MCR2,                // マクロ2
+  MCR3,                // マクロ3
+  MCR4,                // マクロ4
+  MCR5,                // マクロ5
+  WN_SCLN,              // タップでJISの「:」  シフトでJISの「;」 (Windows)
+  DM_PLY1,
+  DM_PLY2,
+  DM_REC1,
+  DM_REC2,
+  DM_RSTP,
+};
+
+// Use Dynamic macro
+
+// Fillers to make layering more clear
+#define LOWER   LT(_LOWER, KC_MHEN)    // タップで無変換     ホールドでLower
+#define RAISE   LT(_RAISE, KC_HENK)    // タップで変換       ホールドでRaise
+#define GUI_ESC GUI_T(KC_ESC)          // タップでESC        ホールドでGUI
+#define SFT_BS  SFT_T(KC_BSPC)         // タップでBackSpace  ホールドでSHIFT
+#define CTL_ENT CTL_T(KC_ENT)          // タップでEnter      ホールドでCTRL
+#define CTL_TAB CTL_T(KC_TAB)          // タップでTAB        ホールドでCTRL
+#define SFT_DEL SFT_T(KC_DEL)          // タップでDELETE     ホールドでSHIFT
+#define ALT_LBR ALT_T(JP_LBRC)         // タップで[          ホールドでALT
+#define SFT_RBR SFT_T(JP_RBRC)         // タップで]          ホールドでSHIFT
+#define SFT_SPC S(KC_SPC)              // Shift + Space
+#define WN_CAPS S(KC_CAPS)             // Caps Lock           (Windows)
+#define GAME    TO(_GAME)              // _GAMEレイヤーへ移動
+#define ADJUST  TO(_ADJUST)            // _ADJUSTレイヤーへ移動
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Qwerty配列
+   * ,----------------------------------.             ,----------------------------------.
+   * |   Q  |   W  |   E  |   R  |   T  |             |   Y  |   U  |   I  |   O  |   P  |
+   * |------+------+------+------+------|             |------+------+------+------+------|
+   * |   A  |   S  |   D  |   F  |   G  |             |   H  |   J  |   K  |   L  |   ;  |
+   * |------+------+------+------+------|             |------+------+------+------+------|
+   * |   Z  |   X  |   C  |   V  |   B  |             |   N  |   M  |   ,  |   .  |   /  |
+   * |------+------+------+------+------+-------------+------+------+------+------+------|
+   * |Alt/[ |WINESC|   ~  |Sft/Bs|Lower |CtlEnt|Space |Raise |CtrlTb|   -  |   '  |Sft/] |
+   * `-----------------------------------------------------------------------------------'
+   */
+  [_QWERTY] = LAYOUT( \
+      KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                        KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    \
+      KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                        KC_H,    KC_J,    KC_K,    KC_L,    WN_SCLN, \
+      KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                        KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, \
+      ALT_LBR, GUI_ESC, JP_TILD, SFT_BS,  LOWER,   CTL_ENT,  KC_SPC,   RAISE,   CTL_TAB, KC_MINS, JP_QUOT, SFT_RBR  \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
OK.
It has changed.